scootapi/client/smoke_test_cmd.go

```diff
@@ -10,57 +10,62 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type smokeTestCmd struct{}
+type smokeTestCmd struct {
+	numJobs  int
+	numTasks int
+	timeout  time.Duration
+}
 
 func (c *smokeTestCmd) registerFlags() *cobra.Command {
 	r := &cobra.Command{
 		Use:   "run_smoke_test",
 		Short: "Smoke Test",
 	}
+	r.Flags().IntVar(&c.numJobs, "num_jobs", 100, "number of jobs to run")
+	r.Flags().IntVar(&c.numTasks, "num_tasks", -1, "number of tasks per job")
+	r.Flags().DurationVar(&c.timeout, "timeout", 180*time.Second, "how long to wait for the smoke test")
 	return r
 }
 
 func (c *smokeTestCmd) run(cl *simpleCLIClient, cmd *cobra.Command, args []string) error {
 	fmt.Println("Starting Smoke Test")
 
-	numJobs := 100
-
+	// TODO(dbentley): migrate away from positional args to flags
```

Comment:
I don't know what scripts people have. I've updated the Makefile. Want me to be bold and just delete them now?